ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/web3j/Web3JClient.java

```diff
@@ -167,7 +167,7 @@ private void logExecutionClientError(final Throwable error, final boolean couldB
     if (error instanceof TimeoutException) {
       eventLog.executionClientRequestTimedOut();
     } else {
-      eventLog.executionClientRequestFailed(error, couldBeAuthError);
+      eventLog.executionClientRequestFailed(couldBeAuthError);
```

Comment:
maybe what we want to do is more like
```
if(Throwables.hasCause(error, ConnectException)) {
eventLog.executionClientConnectFailure();
} else {
eventLog.executionClientRequestFailed(error, couldBeAuthError);
}
```

so that unknown errors still get the trace - we know this one specifically is a connection error ,but no guarantee they are all the same thing at that point...